mkdocs.yml

```diff
@@ -0,0 +1,83 @@
+# Site
+site_name: 'Mobius'
+site_description: 'A functional reactive framework for managing state evolution and side-effects.'
+site_author: 'Spotify'
+site_url: 'https://github.com/spotify/mobius'
+
+# Repository
+repo_name: 'spotify/mobius'
+repo_url: 'https://github.com/spotify/mobius'
+edit_uri: 'tree/main/docs/'
+remote_branch: gh-pages
+
+# Navigation
```

Comment:
I made sure to structure the navigation to be exactly the same as you currently have it in the wiki.

| **Wiki Sidebar**                                                                                                      | **MkDocs Nav**                                                                                                      |
|-----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
| ![wiki-sidebar](https://user-images.githubusercontent.com/4332753/158275438-fa0cee1d-3bde-4e7d-aea6-36c655e28607.png) | ![mkdocs-nav](https://user-images.githubusercontent.com/4332753/158275439-38b54cd2-4104-4249-a6d3-ba0ed5e91137.png) |